sdk/core/Azure.Core/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 ### Features Added
 
+- `TokenRequestContext` added the `IsProofOfPossessionEnabled`, `ResourceRequestMethod`, and `ResourceRequestUri` properties to support Proof of Possession tokens.
```

Comment:
Would you please add a link to the PR to be consistent with how other CHANGELOG entries are formatted below?